cpp/kiss_icp/core/VoxelHashMap.hpp

```diff
@@ -58,9 +58,9 @@ struct VoxelHashMap {
     inline void Clear() { map_.clear(); }
     inline bool Empty() const { return map_.empty(); }
     inline Voxel PointToVoxel(const Eigen::Vector3d &point) const {
-        return Voxel(static_cast<int>(point.x() / voxel_size_),
-                     static_cast<int>(point.y() / voxel_size_),
-                     static_cast<int>(point.z() / voxel_size_));
+        return Voxel(static_cast<int>(std::floor(point.x() / voxel_size_)),
```

Comment:
I also like this diff more as it's much more explicit "what's new".  the eigen implementation is one line, but it hides some of the real details of the underlying change of this particular operation

Reply:
I do not agree; the one-liner does not hide anything; it is literally the same words. But in the name of her majesty efficiency, I also like this one more ;) we can merge for me